presto-docs/src/main/sphinx/release/release-0.217.rst

```diff
@@ -0,0 +1,57 @@
+=============
+Release 0.217
+=============
+
+General Changes
+---------------
+
+* Fix poor query planning performance for queries that are submitted when the cluster does not have the minimum required number of workers available.
```

Comment:
@nezihyigitbasi made the suggestion of changing `slow` to `poor query`. I think that make sense since "slow" does not go with performance, only good or bad?